examples/stencil/README.md

```diff
@@ -0,0 +1,146 @@
+# [Stencil](https://stenciljs.com)
+
+For a new Stencil project follow the [getting started](https://stenciljs.com/docs/getting-started) guide.
+
+To add Tailwind to a Stencil project, start by installing `@fullhuman/postcss-purgecss`, `@stencil/postcss`, `autoprefixer`, `cssnano` and `tailwindcss`:
+
+```sh
+npm install @fullhuman/postcss-purgecss @stencil/postcss autoprefixer cssnano tailwindcss --save-dev
+```
+
+Next, set up the PostCSS plugin in stencil.config.ts
+
+```js
+import { Config } from '@stencil/core';
+import { postcss } from '@stencil/postcss';
+import autoprefixer from 'autoprefixer';
+
+const purgecss = require('@fullhuman/postcss-purgecss')({
+  content: ['./src/**/*.tsx', './src/index.html'],
+  defaultExtractor: content => content.match(/[A-Za-z0-9-_:/]+/g) || []
+});
+
+export const config: Config = {
+  globalStyle: 'src/global/app.css',
+  outputTargets: [
+    {
+      type: 'www',
+      serviceWorker: null,
+      baseUrl: 'http://localhost:5000'
+    }
+  ],
+  plugins: [
+    postcss({
+      plugins: [
+        require('tailwindcss')('./tailwind.config.js'),
+        autoprefixer(),
+        ...(process.env.NODE_ENV === 'production'
+          ? [purgecss, require('cssnano')]
+          : [])
+      ]
+    })
+  ]
+};
+```
```

Comment:
I replaced the contents of my `stencil.config.js` with the above JS snippet. But when I the run `npm build, I get the following errors:

```
[ ERROR ]  ./src\global\css\app.css
           Unable to read css import: @import 'tailwindcss/components';

[ ERROR ]  ./src\global\css\app.css
           Unable to read css import: @import 'tailwindcss/base';

[ ERROR ]  ./src\global\css\app.css
           Unable to read css import: @import 'tailwindcss/utilities';

[ ERROR ]  postcss: src/global/css/app.css, line: 1: src/global/css/app.css
           Unknown word

      L1:  ﻿﻿@import 'tailwindcss/base';

```

Any ideas what I might be missing?

Reply:
Turns out that when I replace the contents of `src/global/css/app.css` (I decided to put the file in the `css` subfolder) with this:

```css
@tailwind base;
@tailwind components;
@tailwind utilities;
```

then doing `run npm build` completes successfully. I found the solution [here](https://discuss.emberjs.com/t/postcss-import-problem-with-tailwindcss-v1-0/16595/9).

Reply:
I've updated the example